src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3342,7 +3342,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     <ItemGroup>
       <IntermediateSatelliteAssembliesWithTargetPath Include="$(IntermediateOutputPath)%(EmbeddedResource.Culture)\$(TargetName).resources.dll"
-                                                     Condition="'%(EmbeddedResource.Culture)' != ''">
+                                                     Condition="'%(EmbeddedResource.Culture)' != '' and '$(MSBuildRuntimeType)' != 'Core'">
```

Comment:
Will this change prevent the Copy to output directory target to try to copy the satellite assemblies?